docs/guides/virtual-mcp-servers.md

```
@@ -65,24 +75,29 @@ metadata:
 
 Check that your virtual servers were created successfully:
 
 ```bash
-# List all virtual servers
 kubectl get mcpvirtualserver -A
 ```
 
-## Step 3: Test Virtual Server Access
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)